Makefile.core.mk

```diff
@@ -78,37 +78,29 @@ clean-proto:
 
 # is all of this needed or is this cruft?
 
-# get imported protos to $GOPATH
 get_dep_proto:
-	GO111MODULE=off go get k8s.io/api/core/v1 k8s.io/api/autoscaling/v2beta1 k8s.io/apimachinery/pkg/apis/meta/v1/ github.com/gogo/protobuf/...
+	GO111MODULE=off GOPATH=/tmp go get k8s.io/api/core/v1 k8s.io/api/autoscaling/v2beta1 k8s.io/apimachinery/pkg/apis/meta/v1/ github.com/gogo/protobuf/...
 
-proto_iscp:
-	protoc -I=${GOPATH}/src -I./pkg/apis/istio/v1alpha2/ --proto_path=pkg/apis/istio/v1alpha2/ --gofast_out=pkg/apis/istio/v1alpha2/ pkg/apis/istio/v1alpha2/istiocontrolplane_types.proto
+proto_iscp: get_dep_proto
+	protoc -I=/tmp/src -I=${GOPATH}/src/ -I=/usr/include/protobuf --gogofast_out=${GOPATH}/src ${GOPATH}/src/istio.io/operator/pkg/apis/istio/v1alpha2/istiocontrolplane_types.proto
```

Comment:
the first part looks good.  the ${GOPATH}/src - understand what your after, I 'll need to think a little bit about how we can generalize this.  I just woke up so I am not super sharp atm, but I think we need a system host bindmount on either `${GOPATH}` or `/`.`${GOPATH}`may work for our use case with operator repo, but may not work for every use case.  THe bindmount should be `rshared` - so will need to test if that will work as I expect.

I guess we can start with that, and expand.

Cheers
-steve